application/src/main/java/org/togetherjava/tjbot/commands/github/GitHubReference.java

```diff
@@ -148,6 +154,11 @@ MessageEmbed generateReply(GHIssue issue) throws UncheckedIOException {
             String titleUrl = issue.getHtmlUrl().toString();
             String description = issue.getBody();
 
+            if (description != null && description.length() > MessageEmbed.DESCRIPTION_MAX_LENGTH) {
+                description =
+                        "description length is greater than allowed limit for embed, please visit github instead";
```

Comment:
give the link at least?

Reply:
it's clickable in title i believe
![image](https://github.com/Together-Java/TJ-Bot/assets/61616007/89620f30-3b55-4ae2-b55b-00af3a950c94)

Reply:
not a fan of the text, UX-wise. its too technical, as if written by a coder for other coders - not regular users.

perhaps reword to: `(too long for a preview, visit GitHub)`